tests/test_core_files.py

```diff
@@ -303,3 +303,20 @@ def test_ipa_files_format(mock_pkinit):
 
     for result in results.results:
         assert result.result in (constants.SUCCESS, constants.WARNING)
+
+
+@patch('os.umask')
+def test_umask(mock_umask):
+    mock_umask.return_value = 0o027
```

Comment:
Not to be too pedantic but can you change the test name to test_bad_umask? That's what this does.

Reply:
Not pedantic at all, I just forgot to check